src/runtime/job.cpp

```diff
@@ -105,7 +106,9 @@ struct Job final : public GCObject<Job, Value> {
   bool keep;
   std::string echo;
   std::string stream_out;
+  std::vector<std::string> tee_out;
   std::string stream_err;
+  std::vector<std::string> tee_err;
```

Comment:
Something I realized is that we need to make sure that virtual jobs write to these files.

Also also, I just realized that we can make runners treat these files as outputs of some random virtual job so that they still get hashed and can be overwritten or otherwise read in.

Reply:
Good hit on testing with virtual jobs.  I think we *don't* want to hash these (yet), though -- you mentioned "incidental files" during the brainstorming in a way which made me think they'll allow things like `wake -o` without full `Path` citizenship, which I think will be ideal for these logs once we get that implemented, but for now I think it's better to just drop them from FnOutputs entirely.

Reply:
Yeah we can't hash since you pointed out that we want to tee things out to the same file. The incidental files can be used but it won't panic if someone overwrites a tee file. I think we can add a check that incidental files are not hashed though and then wake --clean will still work.